amida-tech/amida-auth-microservice
# Feature/js terraform vpc

#### What's this PR do?
Adds terraform templates to create a VPC designed for deploying a 3-tier version of a microservice. Includes multi-az RDS in a private subnet.

#### Related JIRA tickets:
DEVOPS-132

#### How should this be manually tested?
Run the terraform scripts according to the instructions provided in the README. Confirm that the VPC deploys as shown in the reference architecture. Confirm that the RDS instance is deployed in a private subnet. Optionally, deploy a microservice with an autoscaler and load balancer to this new VPC.
